ajax/libs/jqgrid/package.json

```diff
@@ -30,5 +30,20 @@
     "name": "Tony Tomov",
     "url": "http://www.guriddo.net",
     "email": "tony@trirand.com"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/tonytomov/jqGrid.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "css/**/ui.jqgrid*.+(css|map)",
+          "js/**/+(jquery.jqGrid|grid|jquery.fmatter|jquery.sortable)*.+(js|map)",
```

Comment:
We also need non-minified jqDnR.js and jqModal.js? If yes, let's combine the next rule with this rule and don't forget to add and commit the files.